plug-ins/ShelfFilter.ny

```diff
@@ -0,0 +1,34 @@
+$nyquist plug-in
+$version 4
+$type process
+$preview linear
+$name (_ "Shelf Filter")
+$debugbutton disabled
+$author (_ "Steve Daulton")
+$release 2.4.0
```

Comment:
Is this line correct?

Reply:
I'm not sure which line you are referring to, but lines 5 to 8 look fine.

The "release" version number refers to the plugin, not to the version of Audacity. It doesn't really matter what number is used so long as each subsequent update is higher than the previous. All of my plugins use the earliest Audacity version that I tested the plugin with as the initial "release" number, and then increments from that point. If there's any change to the plugin, the "release" number should be bumped.

Reply:
I meant line 8 and this satisfies me.